com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+{
+  "app-id": "com.github.wwmm.PulseEffects",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+  "command": "pulseeffects",
+  "rename-desktop-file": "pulseeffects.desktop",
+  "rename-icon": "pulseeffects",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--own-name=com.github.wwmm.pulseeffects",
+    "--persist=.config/PulseEffects",
```

Comment:
Well, in that case it is probably best for you to just pass `--filesystem=home`, I think we (by that I mean you :) ) should discuss this with upstream to keep things in a single location.

My mentioned patch setting the default to our location is still correct though and should be easy to get that upstreamed.